integration-tests/integration/login/login.spec.js

```diff
@@ -0,0 +1,68 @@
+const MenuPage = require('../../pages/menuPage')
+const UnauthorisedPage = require('../../pages/unauthorisedPage')
+
+context('Login functionality', () => {
+  before(() => {
+    cy.clearCookies()
+  })
+
+  beforeEach(() => {
+    cy.task('reset')
+  })
+
+  it('Root (/) redirects to the auth login page if not logged in', () => {
+    cy.task('stubLoginPage')
+    cy.visit('/')
+    cy.url().should('include', 'authorize')
+    cy.get('h1').should('contain.text', 'Sign in')
+  })
+
+  it('Login page redirects to the auth login page if not logged in', () => {
+    cy.task('stubLoginPage')
+    cy.visit('/login')
+    cy.url().should('include', 'authorize')
+    cy.get('h1').should('contain.text', 'Sign in')
+  })
+
+  it('Page redirects to the auth login page if not logged in', () => {
+    cy.task('stubLogin', {})
+    cy.visit('/login')
+    cy.url().should('include', 'authorize')
+    cy.get('h1').should('contain.text', 'Sign in')
+  })
+
+  it.skip('Logout takes user to login page', () => {
```

Comment:
temporarily commented out, using old menu options so need to investigate fixing header